public/src/vue/ListView.vue

```diff
@@ -20,6 +32,35 @@ export default {
     BottomPathObjects,
     ProjectsCarousel,
   },
+  data() {
+    return {
+      search: "",
+    }
+  },
+  computed: {
+    suggestionsList: function() {
+      let workshops = [];
+      this.projects.forEach(project => {
+        workshops.push(project.workshop_type);
+      });
+      return Array.from(new Set(workshops));
+    },
```

Comment:
try:

    suggestionList: function() {
      return Array.from(
        new Set(
          this.projects.map(project => project.workshop_type)
        )
      );
    };

or with spread operator:

    suggestionList: function() {
      return [...new Set(
        this.projects.map(project => project.workshop_type)
      )];
    },